composer.json

```diff
@@ -37,7 +37,7 @@
         "doctrine/mongodb-odm": "^2.2",
         "doctrine/mongodb-odm-bundle": "^4.0",
         "doctrine/orm": "^2.6.4 || ^3.0",
-        "elasticsearch/elasticsearch": "^6.0 || ^7.0",
+        "elasticsearch/elasticsearch": "7.14.0",
```

Comment:
Is this mandatory or can we relax the contraint a bit (at least allowing new patch versions)?